.travis.yml

```diff
@@ -32,3 +32,4 @@ env:
     PACKAGE=merlin
 os:
   - linux
+  - macos
```

Comment:
I believe the plan is to get rid of Travis sooner or later (@trefis ?), maybe we should not add any more load to it ? 

Or at least only on the latest version of OCaml because testing all versions would take ages...